plugins/kobolight.koplugin/main.lua

```diff
@@ -72,6 +72,12 @@ end
 function KoboLight:resetLayout()
     local new_screen_height = Screen:getHeight()
     self.gestureScale = new_screen_height * swipe_touch_zone_ratio.h * 0.8
+    local powerd = Device:getPowerDevice()
+    local scale = (powerd.fl_max - powerd.fl_min) / 20
+    for i = 1, #self.steps, 1
+    do
+        self.steps[i] = math.floor(self.steps[i] * scale)
```

Comment:
Your change makes sense to me, it's not a good experience to having to scroll 10 times to get to maximum value. On the other hand, I think we should keep the fine tune capability so users don't have to do that through the menu. I think we can just change scale calculation to `(powerd.fl_max - powerd.fl_min) / 19` and leave out `self.steps[1]` as `1`. What do you think?